app/components/BuyNBTC.tsx

```diff
@@ -13,6 +13,31 @@ import { suiToMist } from "~/util/util";
 import { useToast } from "~/hooks/use-toast";
 import { pricePerNBTCInSUI } from "~/constant";
 
+const BUY_NBTC_INSTRUCTIONS = [
+	"Click on Connect Sui Wallet button, if not already connected.",
+	"Use the Slush wallet.",
+	`Click "profile" → network → make sure you select testnet.`,
```

Comment:
let's add "sublist":
* Make sure you have testnet Sui tokens:
  *  Click "profile" → network → make sure you select testnet.
  * ...
  * You can also check [alternative faucets](https://docs.sui.io/guides/developer/getting-started/get-coins)

Reply:
Done